internal/host/monitor.go

```diff
@@ -129,7 +133,15 @@ func (m *Manager) clusterHostMonitoring() int64 {
 
 		for _, c := range clusters {
 			inventoryCache := make(InventoryCache)
-			for _, host := range sortHosts(c.Hosts) {
+			sortedHosts, canRefreshRoles := SortHosts(c.Hosts)
+			// DEBUG CODE
```

Comment:
can remove lines 137-143 then?

Reply:
some debug code is still present. uploading a fix